src/json_interface.jl

```diff
@@ -101,32 +102,62 @@ save model constructor, model parameters, and other information to a JSON file.
 * `errors` : the fitting / test errors computed during the fitting 
 * `verbose` : print information about the saving process     
 """
-function save_model(model, filename; 
-                    make_model_args = nothing, 
-                    errors = nothing, 
-                    verbose = true, 
-                    meta = Dict(), )
-
-   D = Dict("model_parameters" => model.ps, 
-            "meta" => meta)
-
-   if isnothing(make_model_args) 
-      if verbose
-         @warn("Only model parameters are saved but no information to reconstruct the model.")
-      end
-   else 
-      D["make_model_args"] = make_model_args
-   end
 
-   if !isnothing(errors)
-      D["errors"] = errors
-   end
 
-   open(filename, "w") do io
-       write(io, JSON.json(D, 3))
-   end
+function find_manifest(start_path = pwd())
+    current_path = start_path
+    while !isempty(current_path)
+        manifest_path = joinpath(current_path, "Manifest.toml")
+        if isfile(manifest_path)
+            return manifest_path
+        end
+
+        current_path = dirname(current_path)
+        if current_path == "/"
+            break  
+        end
+    end
+    return nothing
+end
 
-   if verbose
-      @info "Results saved to file: $filename"
-   end
+function save_model(model, filename;
```

Comment:
please add a `save_project = true` flag to the keyword arguments.

Reply:
Done